src/main/java/com/shopping_app/shoppingApp/model/Request/UserRegisterRequest.java

```diff
@@ -0,0 +1,65 @@
+package com.shopping_app.shoppingApp.model.Request;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.shopping_app.shoppingApp.domain.Address;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import javax.validation.constraints.NotBlank;
+import java.util.Set;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class UserRegisterRequest {
+    @NotBlank
+    private String name;
+    @NotBlank
+    private String email;
+    @NotBlank
+    private String phoneNumber;
+    @NotBlank
+    private String password;
+    private Set<Address> address;
```

Comment:
```suggestion
    @NotBlank
    private String password;
    
    private Set<Address> address;
```
Please be consistent in the formatting of code as well, either use `@NotBlank private String phoneNumber` for all the classes or use what is done in here (preferable is what's suggested here since it's easy to read in this format)

Reply:
sure